build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
I think that could work, but it has a couple of complications - I'd have to wire up restoring packages on non-Windows, which isn't something we do today. I'm happy to go do it, but I'm not sure how much work it'll be. Also, we can't send Core_Root to Helix from non-Windows, since the whole job (tests & Core_Root) has to be sent together, meaning it has to be sent from Windows - so we'd have to create Core_Root on non-Windows, package it & upload it to azure, restore it on Windows, & re-upload it to Azure with the Windows test build.